lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py

```diff
@@ -96,57 +76,24 @@
 import json
 
 from ansible.module_utils.api import basic_auth_argument_spec
-from ansible.module_utils.pycompat24 import get_exception
-from ansible.module_utils.urls import open_url
-
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.six.moves.urllib.error import HTTPError
+from ansible.module_utils.netapp import request, eseries_host_argument_spec
+from ansible.module_utils.pycompat24 import get_exception
 
 HEADERS = {
```

Comment:
As a future improvement, should we add HEADERS to the netapp util module (and give it a self describing name like HTTP_JSON_HEADERS)?  Requests might already have a static for this.